{
    "id": "CVE-2021-38295",
    "cisaKEV": false,
    "reported_exploited": false,
    "exploit_maturity": "poc",
    "counts": {
        "public_exploit_count": 1
    },
    "timeline": {
        "nvd_published": "2021-10-14"
    },
    "exploits": [
        {
            "url": "https://github.com/ProfessionallyEvil/CVE-2021-38295-PoC",
            "name": "A simple Python proof of concept for CVE-2021-38295.",
            "source": "github",
            "date_added": "2021-10-14",
            "exploit_maturity": "poc"
        }
    ]
}